app/scripts/lib/channels/notifier.js

```diff
@@ -18,9 +18,10 @@ define(function (require, exports, module) {
   var EVENTS = {
     COMPLETE_RESET_PASSWORD_TAB_OPEN: 'fxaccounts:complete_reset_password_tab_open',
     DELETE: 'fxaccounts:delete',
+    DEVICE_NAME_CHANGE: 'fxaccounts:device_name_change',
     PROFILE_CHANGE: 'profile:change',
     SIGNED_IN: 'internal:signed_in',
-    SIGNED_OUT: 'internal:signed_out'
+    SIGNED_OUT: 'fxaccounts:logout'
```

Comment:
Also, I just realised the nonsense I was talking with this statement:

> ...so maybe there was already a duplicate call to session destroy in there before...

Obviously, the `fxaccounts:logout` notification wasn't being sent at this point before, so there couldn't have been a duplicate call previously.